eo-runtime/src/test/eo/org/eolang/int-tests.eo

```diff
@@ -43,3 +43,36 @@
         n.eq 3
         minus1.add minus2
         rec (n.sub 1) (minus1.add minus2) minus1
+
+# Checks that division by zero returns an error object
+[] > zeroDivision
+  eq. > @
+    (2.div 0).msg
+    (error "Division by zero is undefined").msg
```

Comment:
@IngeniariusSoftware why not just 

```
"Division by zero is undefined"
```

Reply:
For explicit error indication

Reply:
@IngeniariusSoftware looks weird. I would just compare with a string.

Reply:
@yegor256 no problem, fixed it